_projects/2023/100505771/100505771.Rmd

```diff
@@ -0,0 +1,284 @@
+---
+title: "Mexico's Export Dynamics from 1996 to 2020 by Sector"
+description: |
+  Replication and alternative version of the over time graph sector exports of Mexico by the Growth Lab at Harvard University.
+author: Valeria Contreras
+date: 2023-12-12
+output:
+  distill::distill_article:
+    self_contained: false
+---
+
+##INTRODUCTION
+
+The Atlas of Economic Complexity, developed by the Growth Lab at Harvard University, introduced a comprehensive profile of countries in the year 2019. Where users are invited to embark enter on an interactive exploration, navigating through the economic structures and growth dynamics of over 130 nations. Within this context, our focus turns to México, where we start our visual journey through its economic landscape, observing the details of export dynamics across diverse sectors in the previous years.
+
+This tutorial aims to replicate the existing graph and present an alternative visualization using the identical dataset.
+
+```{r}
+
+```
+
+```{r}
+
+ 
+
+```
+
+The x-axis represents the timeline from 1996 to 2020, offering a chronological journey through the years. Meanwhile, the y-axis measures the current gross export in billions of dollars, serving as a measure for the economic impact of each sector where each sector is distinguished by a unique color.
+
+##REPLICA
+
+Starting with the journey of replication, we start by recreating the above plot and below you will find the complete code along with some insights into the challenges I faced while recreating it.
+
+```{r}
+
+rm(list = ls()) 
+library(ggplot2)
+library(readxl)
+library(dplyr)
+library(png)
+library(grid)
+library(ggthemes)
+library(ggstream)
+library(showtext)
+
+
+exportmex <- read_xlsx("~/Desktop/EXCHANGE/DATA VISUALIZATION/EXPORTS-MEXICO.xlsx")
+
+
+colors <- c("#b33d6d","#2b607c","#7cdada","#7ba2d9","#8c7ad7","#c47cda","#d87b7a","#bb9689","#dab57c","#f4d025","#7cdaa0")
+
+new_order <- c("#b33d6d","#7cdaa0", "#f4d025", "#dab57c", "#bb9689","#d87b7a", "#c47cda", "#8c7ad7","#7ba2d9","#7cdada","#2b607c")
```

Comment:
See the output in the HTML. Lines of code that are too long get out of the page. Restyle all your code please to meet the maximum width. (A maximum width of 80 characters is generally advisable)